### PR TITLE
Add compat data for TimeRanges

### PR DESCRIPTION
**Pull request by mlbrgl**

### CODE REVIEW

File: api/TimeRanges.json [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/TimeRanges",
+        "support": {
+          "webview_android": {
+            "version_added": null
+          },
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/TimeRanges.json]: It's `true` for Android and webview.

File: api/TimeRanges.json [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          },
+          "ie": {
+            "version_added": true
+          },
+          "opera": {
+            "version_added": true
+          },
+          "opera_android": {
+            "version_added": null
```
Comment (jpmedley) [inline api/TimeRanges.json]: `true`

**Comment by mlbrgl** [comment]
@jpmedley did you mean those three keys should be true?
- webview_android
- chrome_android
- opera_android

**Comment by jpmedley** [comment]
Yes.

**Comment by mlbrgl** [comment]
@jpmedley ok thanks, done.

**Comment by connorshea** [comment]
This has a bunch of unrelated changes that shouldn't be included 🤔

**Comment by Elchi3** [comment]
Replaced by https://github.com/mdn/browser-compat-data/pull/2601. Thanks @mlbrgl and all the best for your PCT hike!

**Pull request closed without merging**